README.adoc

```diff
@@ -147,26 +155,28 @@ container needs to be restarted.
 kubernetes.yaml
 [source, yaml, linenums, role='code_column']
 ----
-include::finish/kubernetes.yaml[tags=**;]
+include::finish/kubernetes.yaml[]
 ----
 
 The readiness and liveness probes are configured for the containers running the `system` 
 and `inventory` microservices.
 
-The readiness probes are configured to poll the [hotspot=25 hotspot=66]`/health/ready` endpoint. The readiness probe 
-determines the READY status of the container as seen in the `kubectl get pods` output. The 
-[hotspot=27 hotspot=68]`initialDelaySeconds` field defines how long the probe should wait before it 
+The readiness probes are configured to poll the [hotspot=ready1 hotspot=ready2]`/health/ready` endpoint.
+The readiness probe determines the READY status of the container as seen in the `kubectl get pods` output.
```

Comment:
add comma

The readiness probe determines the READY status of the container, as seen in the kubectl get pods output.